Gems/ROS2/Code/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ ly_add_target(
         PUBLIC
             AZ::AzCore
             AZ::AzFramework
+            AZ::AzToolsFramework
```

Comment:
Sorry, but it is not acceptable to use AZ::AzToolsFramework in game mode. 
It makes the game executable with this Gem activated almost as heavy as Editor.